omegaconf/basecontainer.py

```diff
@@ -256,7 +256,7 @@ def pretty(self, resolve: bool = False, sort_keys: bool = False) -> str:
     @staticmethod
     def _map_merge(dest: "BaseContainer", src: "BaseContainer") -> None:
         """merge src into dest and return a new copy, does not modified input"""
-        from omegaconf import OmegaConf
+        from omegaconf import OmegaConf  # isort:skip
```

Comment:
I never had to do this.
what problem is this solving?

Reply:
These lines was failing when I run `isort . --check` and isort did not manage to fix them. But the common factor is that they are imports inside methods. Buy adding `# isort:skip`, isort passed with out errors.

Reply:
Do nox pass at your system `#isort:skip` ? Do you have a more correct way to do this?

Reply:
The pull request contains the last changes from master. So no rebase is necessary.

Reply:
nox is passing for me on master.
does it not pass for you?

Show me the output of `nox -s lint-3.6` on master for you.

Reply:
```
nox > Running session lint-3.6
nox > Creating virtual environment (virtualenv) using python.exe in .nox\lint-3-6
nox > pip install --upgrade setuptools pip
nox > pip install -r requirements/lint.txt -e .
nox > mypy . --strict
nox > isort . --check
nox > Command isort . --check failed with exit code 1:
ERROR: C:\Users\kf9762\Desktop\tmp\omegaconf\omegaconf\base.py Imports are incorrectly sorted.
ERROR: C:\Users\kf9762\Desktop\tmp\omegaconf\omegaconf\basecontainer.py Imports are incorrectly sorted.
nox > Session lint-3.6 failed.
```

Reply:
Seems to work for me:
```
omry@omry-fedora:~/dev/omegaconf (master)$ git describe --tags
2.0.0rc6-41-gcffc694
omry@omry-fedora:~/dev/omegaconf (master)$ nox -s lint-3.6
nox > Running session lint-3.6
nox > Creating virtual environment (virtualenv) using python3.6 in .nox/lint-3-6
nox > pip install --upgrade setuptools pip
nox > pip install -r requirements/lint.txt -e .
nox > mypy . --strict
nox > isort . --check
nox > black --check .
nox > Session lint-3.6 was successful.
```